packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -50,47 +66,259 @@ const CustomModalContent = styled(Modal.Content)`
   max-height: 100%;
 `;
 
-const RelationModal = ({ open, onToggle }: RelationModalProps) => {
+type RelationModalWrapperProps = Omit<
+  RelationModalProps,
+  | 'setOnConfirm'
+  | 'setIsConfirmationOpen'
+  | 'isFormModified'
+  | 'open'
+  | 'onToggle'
+  | 'setIsModalOpen'
+>;
```

Comment:
I don't think we will need to do this anymore since most or all of these props aren't used anymore